services/mcp-test-server/.env.example

```
@@ -0,0 +1,42 @@
+KEYCLOAK_REALM=local
+MCP_TEST_SERVICE_NAME=mcp-test-server
+MCP_TEST_SERVICE_NAME_HUMAN="MCP Test Server"
+MCP_TEST_SERVICE_NAME_MODEL=mcp-test-server
+MCP_TEST_SERVICE_DESCRIPTION="Deterministic MCP test harness for OAuth + compose validation"
+# Update KEYCLOAK_REALM and OIDC_ISSUER to match your deployment
+OIDC_ISSUER=http://127.0.0.1:5050/auth/realms/local
+OIDC_AUDIENCE=http://127.0.0.1:8770/mcp
+
+# --- Networking & Compose (override as needed) ---
+MCP_TEST_SERVER_PORT=8770
+MCP_TEST_SERVER_BIND_HOST=0.0.0.0
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MCP_TEST_SERVER_BIND_HOST key should go before the MCP_TEST_SERVER_PORT key

(UnorderedKey)